cloud-aws/src/main/resources/templates/aws-cf-stack-freeipa.ftl

```diff
@@ -0,0 +1,236 @@
+        "MinSize" : 0,
+        "MaxSize" : ${group.instanceCount},
+        "DesiredCapacity" : ${group.instanceCount},
+        "Tags" : [ { "Key" : "Name", "Value" : { "Fn::Join" : ["-", [ { "Ref" : "StackName" }, "${group.groupName}"]] }, "PropagateAtLaunch" : "true" },
+                   { "Key" : "owner", "Value" : { "Ref" : "stackowner" }, "PropagateAtLaunch" : "true" },
+                   { "Key" : "Owner", "Value" : { "Ref" : "StackOwner" }, "PropagateAtLaunch" : "true" },
+                   { "Key" : "cb-resource-type", "Value" : "${instance_resource}", "PropagateAtLaunch" : "true" },
+                   { "Key" : "instanceGroup", "Value" : "${group.groupName}", "PropagateAtLaunch" : "true" }]
+      }
+    },
+
+    "AmbariNodeLaunchConfig${group.groupName?replace('_', '')}"  : {
+      "Type" : "AWS::AutoScaling::LaunchConfiguration",
+      "Properties" : {
+        <#if group.ebsOptimized == true>
+        "EbsOptimized" : "true",
+        </#if>
+        <#if group.hasInstanceProfile>
+        "IamInstanceProfile" : "${group.instanceProfile}",
+        </#if>
+        "BlockDeviceMappings" : [
+          {
+            "DeviceName" : { "Ref" : "RootDeviceName" },
+            "Ebs" : {
+              "VolumeSize" : "${group.rootVolumeSize}",
+              "VolumeType" : "gp2"
+            }
+          }
+        ],
+        <#if group.ebsEncrypted == true>
+        "ImageId"        : "${group.encryptedAMI}",
+        <#else>
+        "ImageId"        : { "Ref" : "AMI" },
+        </#if>
+        <#if group.cloudSecurityIds?size != 0>
+        "SecurityGroups" : [ <#list group.cloudSecurityIds as cloudSecurityId>
+                               "${cloudSecurityId}"<#if cloudSecurityId_has_next>,</#if>
+                             </#list>
+                           ],
+        <#else>
+        "SecurityGroups" : [ { "Ref" : "ClusterNodeSecurityGroup${group.groupName?replace('_', '')}" } ],
+        </#if>
+        "InstanceType"   : "${group.flavor}",
+        "KeyName"        : { "Ref" : "KeyName" },
+        <#if group.spotPrice??>
+        "SpotPrice"      : "${group.spotPrice}",
+        </#if>
+        "UserData"       : { "Fn::Base64" : { "Fn::Join" : ["", [ { "Ref" : "CBGateWayUserData"},
+                                                                  { "Ref" : "CBGateWayUserData1"},
+                                                                  { "Ref" : "CBGateWayUserData2"},
+                                                                  { "Ref" : "CBGateWayUserData3"}]] }}
+      }
+    }
+
+    <#if group.cloudSecurityIds?size == 0>,
+    "ClusterNodeSecurityGroup${group.groupName?replace('_', '')}" : {
+      "Type" : "AWS::EC2::SecurityGroup",
+      "Properties" : {
+        "GroupDescription" : "Allow access from web and bastion as well as outbound HTTP and HTTPS traffic",
+        "Tags" : [
+            { "Key" : "cb-resource-type", "Value" : "${securitygroup_resource}" },
```

Comment:
cb-resource-type owner and Owner should be removed these are not required anymore